ai-data/llm-inference/reference-content/llama-2-70b-chat.mdx

```diff
@@ -52,11 +52,11 @@ To perform inference tasks with your Llama-2 deployed at Scaleway, use the follo
 
 ```bash
 curl -s \
--H "X-Auth-Token: <IAM API key>" \
+-H "Authorization: Bearer <IAM API key>" \
 -H "Content-Type: application/json" \
 --request POST \
---url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/" \
---data '{"text_input": "[INST]How can I use large language models to improve my customer service? [/INST]", "max_tokens": 200, "temperature": 0.2, "random_seed": 1, "top_p": 0.9}' | jq -r .text_output
+--url "https://<Deployment UUID>.ifr.fr-par.scw.cloud/v1/chat/completions" \
+--data '{"messages":[{"role": "user","content": "There is a llama in my garden, what should I do?"}], "max_tokens": 200, "top_p": 1, "top_k": 40, "temperature": 1, "stream": false}'
```

Comment:
We support top_p but not top_k.
top_k is part of the triton api but not openai spec. If you know which parameter to map it from I can add it but it's not supported for now

Reply:
Also, since we are going to use model-specific templating, I believe we should enforce the "model" parameter. It will be for now an enum with the values of the models we support today, so the user would get an error detailing which values are accepted if they do not provide the parameter or provide a bad value. WDYT?

Reply:
- ok introduced model name in every code sample
- top_k is indeed not in the OpenAI specs, let's skip for now and see if users ask for it (other open source models outside OpenAI do have top k)